.gitignore

```diff
@@ -27,3 +27,6 @@ override.tf.json
 
 # Include tfplan files to ignore the plan output of command: terraform plan -out=tfplan
 # example: *tfplan*
+
+# folder "testfolder"
+testfolder
```

Comment:
bitte am ende jeder file immer eine freie Zeile das ist bei alten Programmiersprachen so üblich und man kann sagen gehört zum guten ton bei der Programmierung

Reply:
Thank you. In the next pull request I resolved it already because I saw your comment in another project ;-)